src/tools/debugger/entities/SteppingType.java

```diff
@@ -156,19 +189,29 @@ private static void completeHandleFrameSkip(final Suspension susp) {
       If no tags are given, it is assumed the operation is always valid. */
   public final Class<? extends Tags>[] applicableTo;
 
+  /** Stepping operation is only available when in the dynamic scope of the given entity.
+      If no entity types are given, it is assumed the operation is always valid. */
+  public final EntityType[] inScope;
```

Comment:
Technically, we could see a turn also as a dynamic scope, but, it's not necessary since all operations executed by an actor are executed within a turn. So, it's somewhat redundant.
If there would be any code that's executed outside the scope of turns, it would be necessary.
One example would be actor models that handle message receiving differently and do not have a notion of a turn perhaps.